.github/workflows/publish-to-s3.yaml

```diff
@@ -0,0 +1,32 @@
+name: Publish to S3
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  upload:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        role-to-assume: arn:aws:iam::312560106906:role/hubverse-${{ github.event.repository.name }}-githubaction
+        aws-region: us-east-1
+
+    - name: Copy files to S3
+      run: |
+        aws s3 sync ./auxiliary-data s3://hubverse-${{ github.event.repository.name }}/auxiliary-data --delete
```

Comment:
`--delete` specifies that any files removed from the repo are also removed from S3 during the [sync](https://docs.aws.amazon.com/cli/latest/reference/s3/sync.html)